src/tests/createHeadlessForm.test.js

```diff
@@ -1009,30 +1009,112 @@ describe('createHeadlessForm', () => {
     });
 
     it('support "date" field type', () => {
-      const result = createHeadlessForm(schemaInputTypeDate);
+      const { fields, handleValidation } = createHeadlessForm(schemaInputTypeDate);
```

Comment:
Approved, let's go! Thanks a lot Gabriel!

Just one last thing. The MR title can be improved to follow the conventional commits:

- use `date` not `dates` (date is the inputType name)
- the convention is `type(scope): body` not `type(scope) - body`
- The body doesn't need to be redundant.

```diff
- fix(dates) - fix date validation minDate and maxDate
+ fix(date): Validate based on minDate and maxDate
```

Reply:
Also, as this is a fix, it's a patch version, not a minor. So it will be 0.4.2, not 0.5.0.

Reply:
I'll release the patch after I merge this.

Thanks for the review and your patience.